doc/integrator/cluster.rst

```diff
@@ -0,0 +1,23 @@
+.. _integrator_cluster:
+
+Create a cluster
+================
+
+The first think to do to create cluster is to separate the database and the application.
+
+One possibility is to have the same database for all the chains (integration, production).
+An than we can have the same schema for the static data, and on a deployement can copy the 
+main schema (`Clone schema <https://wiki.postgresql.org/wiki/Clone_schema>`_).
+
+For the next steps it's important to know what the objective, the scalability or the
```

Comment:
to figure out if the main goal is the scalability or the robustness of the application.